### ISSUE TITLE
Upstream template-creator plugin declares more than three Codex default prompts

### ISSUE THREAD

**Issue by styler-ai** [styler-ai/ProjectAtlas#235]
## Summary
ProjectAtlas post-release Codex smoke emits repeated manifest warnings for the upstream `template-creator` plugin:

```text
WARN codex_core_plugins::manifest: ignoring interface.defaultPrompt: maximum of 3 prompts is supported path=<CODEX_HOME>/plugins/cache/openai-primary-runtime/template-creator/26.623.12021/.codex-plugin/plugin.json
```

The installed manifest declares 4 default prompts:

```text
defaultPromptCount=4
```

## Expected behavior
A plugin manifest should declare at most 3 `interface.defaultPrompt` entries, matching the Codex manifest limit.

## ProjectAtlas impact
The warning appears during ProjectAtlas smoke runs and can be mistaken for a ProjectAtlas plugin-manifest issue. ProjectAtlas v0.3.15 already declares only 3 short default prompts.

## Acceptance criteria
- Confirm this is upstream/non-ProjectAtlas ownership or route it to the responsible upstream plugin.
- ProjectAtlas release-smoke notes classify this warning as external if it still appears.
- Keep ProjectAtlas plugin manifest checks covering the 3-prompt limit so this does not regress locally.

**Comment by styler-ai** [comment]
Closing as not planned for ProjectAtlas: this warning is for upstream template-creator. ProjectAtlas v0.3.15 already declares exactly three Codex defaultPrompt entries, and cargo test -p projectatlas-cli plugin_installers_require_matching_runtime_version passes while enforcing the max-3 prompt limit.
